utils/sparse.py

```diff
@@ -20,13 +19,13 @@
 
 def _get_model_framework_file(model, path):
     available_files = model.training.default.files
-    transfer_request = 'recipe_type=transfer' in path
+    transfer_request = search("recipe(.*)transfer", path)
```

Comment:
Wouldn't this regex match anything that starts with recipe and ends with transfer; for example would this match

`recipe_stupid=dont_transfer`

Reply:
yes. for now this is the plan, anything which starts with recipe and has word transfer innit qualifies as a condition for transfer request